app/models/course.rb

```diff
@@ -0,0 +1,6 @@
+class Course < JsonApiClient::Resource
```

Comment:
Perhaps we can put this into a `FindAPI` namespace, so it's clearer that it's not a database model and therefore needs some extra attention (rescuing exceptions, making sure we don't make too many requests).